README.md

```diff
@@ -3,6 +3,7 @@
 [![GZip](http://pkg.julialang.org/badges/GZip_0.3.svg)](http://pkg.julialang.org/?pkg=GZip)
 [![GZip](http://pkg.julialang.org/badges/GZip_0.4.svg)](http://pkg.julialang.org/?pkg=GZip)
 [![GZip](http://pkg.julialang.org/badges/GZip_0.5.svg)](http://pkg.julialang.org/?pkg=GZip)
+[![GZip](http://pkg.julialang.org/badges/GZip_0.6.svg)](http://pkg.julialang.org/?pkg=GZip)
```

Comment:
add 0.7, maybe remove 0.3-0.5?

Reply:
Will add 0.7 as soon as I have my laptop back (hopefully in a couple of days).